examples/tmx-exampleapps/SampleBSMPlugin/src/SampleBSMPlugin.cpp

```diff
@@ -0,0 +1,144 @@
+#include "SampleBSMPlugin.h"
```

Comment:
This change was not included in the release notes.  I think this was added for VOICES project for simulation and may be beneficial for others to use for similar purposes.

Reply:
Added a bullet for the Sample plugin changes